integrations/dynatrace/client.py

```diff
@@ -63,10 +63,15 @@ async def get_slos(self) -> AsyncGenerator[list[dict[str, Any]], None]:
     async def _get_entities_from_type(
         self, type_: str
     ) -> AsyncGenerator[list[dict[str, Any]], None]:
+        ENTITY_FIELDS = "firstSeenTms,lastSeenTms,tags"
```

Comment:
why not enabling it as part of the resource selector?

Reply:
They are fields defined in the blueprint itself. If they are put in the resource selector, there is the possibility of users overriding it. By putting it here, we ensure they are included automatically. They are not meant to be customized.

Reply:
Wdym? The blueprint is only a default option for modeling the data
The customer can and will change the blueprint to his needs
Therefore we need to allow him to pick which data he wants

Reply:
If that's the case, which do you suggest:
- Take this out and make it as a default option on the selector
- Take this out, take the corresponding fields out in the default blueprint and add it to the selector.
- Any other option...

Reply:
First option should be good